option/internaloption/internaloption.go

```diff
@@ -106,3 +107,15 @@ type enableJwtWithScope bool
 func (w enableJwtWithScope) Apply(o *internal.DialSettings) {
 	o.EnableJwtWithScope = bool(w)
 }
+
+// WithCredentials returns a client option to specify authenticates API calls.
```

Comment:
Seems like this sentence is a fragment-- should it be something like "a client option to specify credentials which will be used to authenticate API calls"?

Reply:
good catch!